### PR TITLE
[MNOE-141] Improve user menu

### PR DESCRIPTION
**Pull request by clemthenem**
Changed the style of the user menu following the design of the maestrano (legacy) user menu.
Added a couple of minor changes.

### CODE REVIEW

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.less [hunk 1]
```diff
@@ -27,58 +29,127 @@ dashboard-company-selectbox {
...
+        .company {
+          width: 100%;
+          display: block;
+          white-space: nowrap;
+          overflow: hidden;
+          text-overflow: ellipsis;
+        }
+        .name {
+          font-weight: bold;
+          color: #17262d;
```
Comment (ouranos) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.less]: variable?

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 2]
```diff
@@ -5,15 +5,13 @@
...
       <div class="title">
         <span class="name ng-binding">{{ selectBox.user.name }} {{ selectBox.user.surname }}</span><br>
-        <span class="company ng-binding" tooltip-placement="bottom" tooltip="{{ selectBox.organization.name }}">{{ selectBox.organization.name }}</span>
-      </div>
-      <div class="expand-menu" ng-switch="selectBox.isClosed">
-        <i class="fa fa-chevron-down ng-scope" ng-switch-when="true"></i>
-        <i class="fa fa-chevron-up ng-scope" ng-switch-when="false"></i>
+        <span class="company ng-binding">{{ selectBox.organization.name }}</span>
       </div>
+      <i class="expand-menu fa" ng-class="{true:'fa-chevron-down', false:'fa-chevron-up'}[selectBox.isClosed]"></i>
```
Comment (ouranos) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: Might be easier to read with a ternary operator?

```
ng-class="'selectBox.isClosed ? fa-chevron-down' : 'fa-chevron-up'"
```

Comment (clemthenem) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: Absolutely, forgot about this easier way. Thanks!

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.less (hunk 1 above)
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.less]: variable

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 3]
```diff
@@ -4,35 +4,41 @@
...
   <div class="wrapper">
-    <div class="box-header">
-      <span class="title">{{ selectBox.user.name }} {{ selectBox.user.surname }}</span>
-      -
-      <span class="title">{{ selectBox.organization.name }}</span>
-      <a type="button" class="btn switch-company" ng-click="selectBox.toggle()">
-        <i ng-class="!selectBox.isClosed && 'active'"></i>
-      </a>
+    <div class="box-header" ng-click="selectBox.toggle()">
+      <img ng-src="{{selectBox.user.avatar_url}}" src="{{selectBox.user.avatar_url}}" />
```
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: You don't need the `src` only the `ng-src`.

https://docs.angularjs.org/api/ng/directive/ngSrc

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 4]
```diff
@@ -4,35 +4,41 @@
...
+      <div class="title">
+        <span class="name ng-binding">{{ selectBox.user.name }} {{ selectBox.user.surname }}</span><br>
```
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: Remove `ng-binding`

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 5]
```diff
@@ -4,35 +4,41 @@
...
+      <i class="expand-menu fa" ng-class="selectBox.isClosed? 'fa-chevron-down': 'fa-chevron-up'"></i>
     </div>
 
     <div class="box-content scrollbar-hidden" uib-collapse="selectBox.isClosed">
       <ul class="list-unstyled">
-        <li ng-repeat="organization in selectBox.user.organizations | orderBy:'name'" >
-          <a href="" ng-click="selectBox.changeTo(organization)">
-            <i class="fa" ng-class="organization === selectBox.organization ? 'fa-dot-circle-o' : 'fa-circle-o'"></i> {{organization.name}}
+        <li ng-repeat="organization in selectBox.user.organizations | orderBy:'name'"  ng-click="selectBox.changeTo(organization)">
+          <a>
```
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: Please set the `ng-click` on the `a`

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 6]
```diff
@@ -4,35 +4,41 @@
...
-        <a href="" ng-click="selectBox.createNewOrga()"><small>{{ 'mno_enterprise.templates.dashboard.menu.new_company' | translate }}</small></a>
+        <button type="button" class="btn btn-warning" ng-click="selectBox.createNewOrga()">
+          <a>{{ 'mno_enterprise.templates.dashboard.menu.new_company' | translate }}</a>
+        </button>
       </div>
-      <div class="admin-link" ng-if="selectBox.user.admin_role">
-        <a href="/admin/"><small>{{ 'mno_enterprise.templates.dashboard.menu.access_admin_panel' | translate }}</small></a>
+      <div ng-if="selectBox.user.admin_role">
+        <hr>
+        <a href="/admin/" class="default-link">
```
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: While we are working on this, I think there is a constant now that we could reuse for the admin path.

**Pull request merged**